kuma/static/styles/components/wiki/contributor-avatars.styl

```diff
@@ -1,18 +1,27 @@
 /* contributor avatars at top of articles */
+
+$avatar-max-width = 20px;
+$avatar-margin = 5px;
+$avatar-limit = 6;
+
 .contributor-avatars {
-    set-smaller-font-size();
+    bidi-value(float, right, left);
+    position: relative;
+    max-width: ($avatar-limit * ($avatar-max-width + $avatar-margin));
+    margin-top: 23px;
+    bidi-style(margin-left, $grid-spacing, margin-right, 0);
+    margin-bottom: $grid-spacing;
     color: #777;
-    padding: 10px 10px $content-vertical-spacing;
-    border-bottom: 1px solid rgb(241, 241, 241);
-    border-top: 1px solid rgb(241, 241, 241);
-    background: none repeat scroll 0 0 rgba-fallback(rgba(234, 239, 243, 0.4));
-    margin-top: -20px;
-    margin-bottom: 20px;
+    set-smaller-font-size();
+
+    .from-search &,
```

Comment:
It over-laps with the button on the right and it was complicated to fix because the <h1> starts floating. I can fix it if we think it's worth our time but I made a value judgement that is is not, with the search interface showing it makes some amount of sense to de-clutter other parts of the interface.